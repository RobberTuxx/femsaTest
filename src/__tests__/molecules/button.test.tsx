import React from 'react';
import renderer from 'react-test-renderer';
import Button from "../../components/molecules/button";
it('render button', ()=> {
    const component = renderer.create(<Button text={'hello world!'} big onPress={()=>{}}/>).toJSON()
    expect(component).toMatchSnapshot()
})
it('has 0 child', ()=> {
    const component = renderer.create(<Button text={'hello world!'} big onPress={()=>{}}/>).toJSON()
    expect(component.children.length).toBe(1)
})
