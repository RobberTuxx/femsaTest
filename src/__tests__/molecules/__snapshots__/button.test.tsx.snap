// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render button 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#334FFA",
      "borderRadius": 10,
      "height": 78.5,
      "justifyContent": "center",
      "marginHorizontal": 11.5,
      "opacity": 1,
      "width": 673.5,
    }
  }
>
  <Text
    style={
      {
        "color": "#ffffff",
        "fontFamily": "Avenir",
        "fontSize": 22,
        "fontWeight": "800",
      }
    }
  >
    hello world!
  </Text>
</View>
`;

exports[`render product name big 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#334FFA",
      "borderRadius": 10,
      "height": 78.5,
      "justifyContent": "center",
      "marginHorizontal": 11.5,
      "opacity": 1,
      "width": 673.5,
    }
  }
>
  <Text
    style={
      {
        "color": "#ffffff",
        "fontFamily": "Avenir",
        "fontSize": 22,
        "fontWeight": "800",
      }
    }
  >
    hello world!
  </Text>
</View>
`;
